src/device-registry/utils/generate-filter.js

```
@@ -22,6 +22,41 @@ const isLowerCase = (str) => {
   return str === str.toLowerCase();
 };
 //startTime=2022-12-20T10:34:15.880Z
+
+function applyAQIFilter(filter, index) {
+  if (!index) {
+    delete filter["values.pm2_5.value"];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)